### ISSUE TITLE
`not [browser] all` sometimes not parsable

### ISSUE THREAD

**Issue by thany** [browserslist/browserslist#749]
When including in a query `not op_mini all` it correctly excludes all versions of Opera Mini. When including `not KaiOS all` it complains with an `Unknown browser query` error. Same for `not kaios all`. At first I was like "what the hecko is the correct name for this KaiOS thing then??" But then, when I saw someone else exclude browsers differently, I changed it to `not kaios <= 100` and it works, and does what I need it to. But why do two browsers require a different syntax to do the exact same thing?

The problem is, because of the above, [this list](https://github.com/browserslist/browserslist#browsers) feels inaccurate, and I can't seem to find another good, solid, comprehensive list of which browser names are supported. And somehow for Opera Mini I can use the `all` qualifier, but not for every browser. And for other browser, I can use the ` <= 100` qualifier, but not for every browser. This is weird, and inconsistent.

To sum up:

| Command | Works? |
|---|---|
| `op_mini all` | ⭕ Yes |
| `kaios all` | ❌ No |
| `and_qq all` | ❌ No |
| `and_uc all` | ❌ No |
| `op_mini <= 100` | ❌ No |
| `kaios <= 100` | ⭕ Yes |
| `and_qq <= 100` | ⭕ Yes |
| `and_uc <= 100` | ⭕ Yes |

And why do I see a mishmash of OperaMini and op_mini, OperaMobile and op_mob, and possibly others. Can this please be kept consistent? I don't care what the actual names are, but using multiple names for the same browser, especially in the very same documentation file, is no less than confusing, tbh.

**Comment by ai** [comment]
`all` is not a keyword. It is a version:

```sh
$ npx browserslist 'last 1 op_mini version'
op_mini all
```

This is inconsistency in Can I Use internal data and I can’t change it.

We can add `all` query for other browsers or we can change docs.

**Comment by ai** [comment]
I added notes to docs and [website](https://browsersl.ist/): 2543e0f https://github.com/browserslist/browsersl.ist/commit/e091427013c28aa883cf6329cdae4b3c31a0fdf2

**Comment by gian1200** [comment]
Regarding this, is it correct to asume that "not [browser] >0" can be a workaround for the lack of "not [browser] all" on all browsers?

**Comment by ai** [comment]
I think `not [browser] >0` should work for most of the browsers (not sure about `op_mini`)

**Comment by thany** [comment]
If it doesn't work consistently, you're basically confirming that this bug exists. Shouldn't it be fixed then?

**Comment by ai** [comment]
@thany send PR, we will accept it.

**Comment by thany** [comment]
That's kind of a weak argument, tbh. Default response in opensource projects when author has neither interest nor intention of fixing a bug.

Besides, I'm not longer affected by this, so my motivation is reduced anyway. iow, someone else send PR, and I will co-accept it 😀

**Comment by ai** [comment]
@thany 

> Default response in opensource projects when author has neither interest nor intention of fixing a bug.

No. Collaboration is a core idea of open source.

Open source is not about having free product and free support. It is about that we are all working together.

If you have an issue, and it bothers you, you should send PR as a good open source user. Because you are the same developer as I am.
